index.js

```diff
@@ -422,6 +422,8 @@ class Raspi extends EventEmitter {
     const pinInstance = this[getPinInstance](this.normalize(pin));
     if (pinInstance.mode === INPUT_MODE && value === HIGH) {
       this[pinMode]({ pin, mode: INPUT_MODE, pullResistor: PULL_UP });
+    } else if (pinInstance.mode === INPUT_MODE && value === LOW) {
```

Comment:
@nebrius Ah, sorry if I've done things the wrong way round. Johnny-Five + Raspi-IO is my go-to for doing physical computing on the Pi so thanks for that! 

This PR (and the [soft PWM one](/nebrius/raspi-io/pull/74)) mean that I can use the built-in Johnny-Five components like `Button` and `RGB LED`s with more of the Pi's GPIO pins without needing extra resistors.